### PR TITLE
Gamefi Vaults issues fix 

### PR DESCRIPTION
**Pull request by kojiadrianojr**
Reference issue #230

### CODE REVIEW

**Review by Emman1598 — APPROVED** [review]

File: src/views/Gamefi/Gamefi.tsx [hunk 1]
```diff
@@ -149,7 +149,7 @@ const Gamefi: React.FC = () => {
     const render = (type) => {
       switch (type) {
         case 'RENDER_ENDED':
-          return (
+          return stakedMemoized.inactiveFarms.length !== 0 && stakedMemoized.inactivePools.length !== 0 ? (
             <>
```
Comment (medardm) [inline src/views/Gamefi/Gamefi.tsx:153]: iwasan ang nested conditions using ternary operator kasi nagiging confusing

try mo si '&&' approach

**Review by medardm — CHANGES_REQUESTED** [review]
Please see comments

File: src/views/Gamefi/Gamefi.tsx (hunk 1 above)
Comment (kojiadrianojr) [reply src/views/Gamefi/Gamefi.tsx:153]: Di man yan kuys nested hahah. Dual lang
First: check if both meron para magrender isa lang. 

Diba meron ata ditong linting maga show naamn if nested or maga throw error

Comment (kojiadrianojr) [reply src/views/Gamefi/Gamefi.tsx:153]: Mag render isa lang na error message ( not available component ) 


Or alisin ko nalang as is na pag both wala, dalawa rin error message ?

**Pull request merged**